src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -47,6 +47,7 @@ namespace AppInstaller::CLI::Execution
         SearchResult,
         SourceList,
         Manifest,
+        SourceId,

```

Comment:
>SourceId [](start = 8, length = 8)

This value can easily be retrieved from Source; why do we need to copy it into the context?

Reply:
I see better now, but I think that the better solution might still be to have the IPackageVersion in the context, and create an IPackageVersion::GetSource method.

---
In reply to: [510503004](https://github.com/microsoft/winget-cli/pull/625#discussion_r510503004) [](ancestors = 510503004)